modules/avatar/react/lib/Avatar.tsx

```diff
@@ -28,17 +27,9 @@ export interface AvatarProps extends React.ButtonHTMLAttributes<HTMLButtonElemen
    * The url of the users avatar photo
    */
   url?: string;
-  /**
-   * An event handler function that gets called when the avatar is clicked
-   */
-  onClick?: (e: React.SyntheticEvent) => void;
-  /**
-   * Ref of button that the styled component renders.
-   */
-  buttonRef?: React.Ref<HTMLButtonElement>;
 }
 
-const Container = styled('button', {
+const Container = styled('div', {
   shouldForwardProp: prop => isPropValid(prop) && prop !== 'size',
 })<Omit<AvatarProps, 'altText'>>(
```

Comment:
Nit: Can you change this to `<Pick<AvatarProps, 'size'>>`?

Reply:
sure